apps/ui/src/views/Space/Editor.vue

```diff
@@ -219,6 +216,14 @@ const proposalMaxEnd = computed(() => {
   );
 });
 
+const {
+  data: propositionPower,
+  isPending: isPropositionPowerPending,
+  isFetching: isPropositionPowerLoading,
```

Comment:
I'd only keep `isPending` as it will show only if no data is available at all. `isFetching` will be true even if cached data is available so if I navigate back and forth it will show loading indicator. Normally with vue-query you don't care about if query is fetching or not, only if it's pending.